flow/storage/elasticsearch/elasticsearch.go

```diff
@@ -154,13 +154,16 @@ const rawPacketMapping = `
 // ElasticSearchStorage describes an ElasticSearch flow backend
 type ElasticSearchStorage struct {
 	client *esclient.ElasticSearchClient
+	perf   map[string]*common.PerfCounterIntRate
```

Comment:
I would prefer to have a global maybe in the perfcounter package itself so that less intrusion in the current code

Reply:
you perfer to have a kind of global tree ? (and remove the perf per instance ?)
rootPerf["storage.elasticsearch"]["StoreFlows"].Prolog(...